communis-services/src/main/resources/application.yml

```diff
@@ -3,6 +3,7 @@ spring:
     mongodb:
       uri: mongodb://localhost:27017
       database: communis
+      auto-index-creation: true
```

Comment:
Does this tie back to [autoIndexCreation](https://github.com/jmathewt/communis/pull/6/files#diff-f4fa4cff2f5c1202bf78b1ea47e868f3R34)?

Reply:
Yeah. We want auto index creation